src/rgw/rgw_auth.cc

```diff
@@ -617,6 +617,14 @@ bool rgw::auth::LocalApplier::is_identity(const idset_t& ids) const {
 	       (id.get_id() == user_info.user_id.id)) {
       return true;
     }
+    for (auto subuser : user_info.subusers) {
+      std::string user = user_info.user_id.id;
+      user.append(":");
+      user.append(subuser.second.name);
+      if (user == id.get_id()) {
+        return true;
+      }
```

Comment:
I think there should be a check for tenant also here, similar to the tenant check for a user.

Reply:
Done :heavy_check_mark: